### PR TITLE
Fix inner utxo ops error in ComputeTransactionMetadata

### PR DESCRIPTION
**Pull request by lazynina**

### CODE REVIEW

**Comment by lazynina** [comment]
* **#1234** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1234?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `feature/proof-of-stake`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @lazynina and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/legacy_mempool.go [hunk 1]
```diff
@@ -2077,23 +2094,6 @@ func ComputeTransactionMetadata(
...
-				break
-			}
-		}
-		if !transactorPublicKeyFound {
-			txnMeta.AffectedPublicKeys = append(txnMeta.AffectedPublicKeys, &AffectedPublicKey{
-				PublicKeyBase58Check: txnMeta.TransactorPublicKeyBase58Check,
-				Metadata:             "TransactorPublicKeyBase58Check",
-			})
-		}
-	}
```
Comment (lazynina) [inline lib/legacy_mempool.go:2096]: This got duplicated somehow. See lines right above.

Comment (poolcoke) [reply lib/legacy_mempool.go:2096]: Weird, not sure how this got duplicated.

**Review by poolcoke — APPROVED** [review]
All makes sense and looks good to me!

**Review by tholonious — APPROVED** [review]

**Comment by lazynina** [comment]
### Merge activity

* **Apr 17, 10:21 AM EDT**: @lazynina started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/deso-protocol/core/1234).

**Pull request merged**